tgui/packages/tgfont/README_ICON_TUTORIAL.txt

```diff
@@ -0,0 +1,26 @@
+The following is the process to implement your own icon using an svg.
+
+If you plan on making your own SVG, i suggest Inkscape, as it is free and pretty powerful for vector graphics.
+
+1. Get whatever SVG you plan on using and put it in the ..\tgstation\tgui\packages\tgfont\icons folder.
```

Comment:
You should atomize your PR’s, this is not very relevant to adding the quirk

Reply:
> You should atomize your PR’s, this is not very relevant to adding the quirk

Absolutely is relevant because if anyone wants to make a quirk in the future this is the only way to add a custom icon to the quirk.